pkg/security/tests/filters_test.go

```diff
@@ -21,6 +21,7 @@ import (
 
 	"github.com/DataDog/datadog-agent/pkg/security/model"
 	"github.com/DataDog/datadog-agent/pkg/security/probe"
+	sprobe "github.com/DataDog/datadog-agent/pkg/security/probe"
```

Comment:
I think we should keep only one import to "github.com/DataDog/datadog-agent/pkg/security/probe"

Reply:
yes, auto import of the IDE :) thanks for pointing this